content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -1033,6 +1033,24 @@ and legacy in-tree cloud providers), it sets this annotation on the Node to deno
 set from the command line flag (`--node-ip`). This IP is verified with the cloud provider as valid
 by the cloud-controller-manager.
 
+### volume.alpha.kubernetes.io/node-affinity (alpha) {#volume-alpha-kubernetes-io-node-affinity}
+
+Type: Annotation
+
+Example: `volume.alpha.kubernetes.io/node-affinity: "constrain-with-nodes"`
+
+Used on: Node
+
+The `volume.alpha.kubernetes.io/node-affinity` annotation is used to specify node affinity rules for a 
+PersistentVolume (PV) in Kubernetes. It allows you to restrict the scheduling of Pods that use the corresponding 
+PersistentVolume to specific nodes in the cluster, based on [node labels](/docs/concepts/storage/persistent-volumes/#selector).
```

Comment:
```suggestion
The `volume.alpha.kubernetes.io/node-affinity` annotation was used to specify experimental node affinity
rules for PersistentVolumes (PV) in Kubernetes. Volume _node affinity_ allows you to restrict the
scheduling of Pods that use the corresponding PersistentVolume so that the Pods can only bound to
appropriate nodes in your cluster, based on the {{<glossary_tooltip text="labels" term_id="label">}}
that are set on those nodes.

You should now use the [`nodeAffinity` field](/docs/concepts/storage/persistent-volumes/#node-affinity) instead.
```